buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -618,7 +618,26 @@ def set_build_variables() {
 def set_sdk_variables() {
     DATESTAMP = get_date()
     SDK_FILE_EXT = SPEC.contains('zos') ? '.pax' : '.tar.gz'
-    SDK_FILENAME =  "OpenJ9-JDK${SDK_VERSION}-${SPEC}-${DATESTAMP}${SDK_FILE_EXT}"
+    // Get full version from tag (eg. 11.0.8+10) and replace + with _
+    FULL_SDK_VERSION = sh (
+        script: "sed -n -e 's/+/_/g; s/^OPENJDK_TAG := //p' < closed/openjdk-tag.gmk",
+        returnStdout: true
+    ).trim()
+    echo "FULL_SDK_VERSION:'${FULL_SDK_VERSION}'"
+
+    // Use template from variable file if provided otherwise default
+    SDK_FILENAME = get_value(VARIABLES.misc.sdk_filename_template, BUILD_IDENTIFIER) ?: get_value(VARIABLES.misc.sdk_filename_template, 'default')
+    echo "SDK_FILENAME_TEMPLATE:'${SDK_FILENAME}'"
```

Comment:
Shouldn't `SDK_FILENAME_TEMPLATE` be  `SDK_FILENAME `?

Reply:
Since I am echoing twice, I called the pre-processed version 'template', I originally had 2 variables but then combined them since I don't need to keep the template version.

Reply:
This is confusing since SDK_FILENAME_TEMPLATE is not a variable. 
`echo "SDK_FILENAME (template): '${SDK_FILENAME}'"`

Reply:
I would rather keep both echos so we can see the variable/template being processed. I will rename the first variable to include template so it's more clear.